packages/components/modules/messages/MessagesList/MessagesGroup/index.tsx

```diff
@@ -125,41 +126,47 @@ const MessagesGroup: FC<MessagesGroupProps> = ({
     <Box display="flex" flexDirection="column" sx={{ paddingTop: 1 / 2, paddingRight: 2 }}>
       {renderUnreadMessagesDivider(messageIndex)}
       {renderDateOnTopOfMessagesGroup(messageIndex)}
-      <Box sx={{ display: 'flex', flexDirection: 'row', alignSelf: flexAlignments, width: '100%' }}>
-        {canShowAvatar && (
-          <Box paddingRight="12px">
-            <AvatarWithPlaceholder
-              className="self-start justify-self-center"
-              width={32}
-              height={32}
-              src={message?.profile?.image?.url}
-              sx={{ border: 'none' }}
-            />
-          </Box>
-        )}
+      {message.messageType === MESSAGE_TYPE.system ? (
```

Comment:
maybe extract the ternary into a function/component? Just in case we need to add more messages types in the future

Reply:
@tsl-ps2 can we address this as well?